static/js/vue-cdr-access/src/components/full_record/abstract.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+    <p v-if="truncateAbstract" class="abstract">{{ truncatedAbstractText }}... <a class="abstract-text" @click.prevent="toggleAbstractDisplay()"
+                                                                                  href="#">{{ abstractLinkText }}</a></p>
+    <p v-else class="abstract">{{ recordData.briefObject.abstractText }}</p>
+</template>
+
+<script>
```

Comment:
I think you might need a `props: {` section so that this component can receive a briefObject. I think you can also safely just pass in `briefObject` and not the wrapping `recordData` if you don't actually need anything from outside briefObject.